ci/steps/brew-bump.sh

```diff
@@ -77,36 +70,33 @@ main() {
   chmod +x "$PATH_TO_GIT_ASKPASS"
 
   # Make sure the git-askpass.sh file is executable
-  if is_executable "$PATH_TO_GIT_ASKPASS"; then
+  if ! is_executable "$PATH_TO_GIT_ASKPASS"; then
     echo "$PATH_TO_GIT_ASKPASS is not executable."
     ls -la "$PATH_TO_GIT_ASKPASS"
     exit 1
   fi
 
+  # NOTE: we need to make sure our fork is up-to-date
+  # otherwise, brew bump-formula-pr will use an
+  # outdated base
+  echo "Pushing changes to $GITHUB_USERNAME/$REPO fork on GitHub"
   # Export the variables so git sees them
   export HOMEBREW_GITHUB_API_TOKEN="$HOMEBREW_GITHUB_API_TOKEN"
-  export GIT_ASKPASS="$PATH_TO_ASKPASS"
-  git push https://coder-oss@github.com/coder-oss/homebrew-core.git --all
+  export GIT_ASKPASS="$PATH_TO_GIT_ASKPASS"
+  git push "https://$GITHUB_USERNAME@github.com/$GITHUB_USERNAME/$REPO.git" --all
 
   # Find the docs for bump-formula-pr here
   # https://github.com/Homebrew/brew/blob/master/Library/Homebrew/dev-cmd/bump-formula-pr.rb#L18
-  local output
-  if ! output=$(brew bump-formula-pr --version="${VERSION}" code-server --no-browse --no-audit 2>&1); then
-    if [[ $output == *"Duplicate PRs should not be opened"* ]]; then
-      echo "$VERSION is already submitted"
-    else
-      echo "$output"
-      exit 1
-    fi
-  fi
+  # local output
+  brew bump-formula-pr --version="${VERSION}" code-server --no-browse --no-audit
 
   # Clean up and remove homebrew-core
   popd
-  rm -rf homebrew-core
+  rm -rf "$REPO"
 
-  # Make sure homebrew-core is removed
-  if directory_exists "homebrew-core"; then
-    echo "rm -rf homebrew-core failed."
+  # Make sure $REPO is removed
+  if directory_exists "$REPO"; then
+    echo "rm -rf $REPO failed."
     ls -la
   fi
```

Comment:
I am thinking we might want to remove this, for example if I ran it locally could be weird that my clone gets removed, plus since we clone outside this script I think any cleanup could happen outside the script as well, but in this case it will get automatically cleaned up when the job ends.